test/FastTests/Voron/Tables/RavenDB_22226.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Sparrow.Server;
+using Voron;
+using Voron.Data.Tables;
+using Voron.Impl;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace FastTests.Voron.Tables
+{
+    public unsafe class RavenDB_22226 : TableStorageTest
+    {
+        public static readonly Slice IndexName;
+
+        public RavenDB_22226(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        static RavenDB_22226()
+        {
+            using (StorageEnvironment.GetStaticContext(out var ctx))
+            {
+                Slice.From(ctx, "DynamicKeyIndex", ByteStringType.Immutable, out IndexName);
+            }
+        }
+
+        [Fact]
```

Comment:
Please use `RavenFact`. 

Also please note that the tests run got aborted - https://jenkins.hibernatingrhinos.com/job/PR_Tests/10396/

Reply:
done, I guess the hang is not related, but lets give it another test run